src/main/deltachat/backup.ts

```diff
@@ -14,8 +14,29 @@ const log = getLogger('main/deltachat/backup')
 import SplitOut from './splitout'
 import { DeltaChatAccount } from '../../shared/shared-types'
 export default class DCBackup extends SplitOut {
-  export(dir: string) {
-    this._dc.importExport(C.DC_IMEX_EXPORT_BACKUP, dir, undefined)
+  async export(dir: string) {
+    this._dc.stopIO()
+    try {
+      await this._internal_export(dir)
+    } catch (err) {
+      this._dc.startIO()
+      throw err
+    } finally {
+      this._dc.startIO()
```

Comment:
Why put it in a finally statement? Is somehow confusing to me. I guess it won't get executed if you throw an err in the catch branch?

Reply:
> I guess it won't get executed if you throw an err in the catch branch?

yes, exactly